src/java.desktop/share/classes/com/sun/imageio/plugins/tiff/TIFFImageWriteParam.java

```diff
@@ -78,7 +78,7 @@
  * <tr>
  * <td>Exif JPEG</td>
  * <td>Exif-specific JPEG compression (see note following this table)</td>
- * <td><a href="http://www.exif.org/Exif2-2.PDF">Exif 2.2 Specification</a>
+ * <td><a href="https://www.cipa.jp/std/documents/e/DC-008-2012_E.pdf">Exif 2.2 Specification</a>
```

Comment:
BTW the link at the other end is EXIF 2.3 .. so you probably want to fix the text too.